content/docs/reference/glossary.md

```diff
@@ -168,6 +208,10 @@ A browser-based graphical interface for managing Neon projects and resources.
 
 The user account that registers and authenticates with Neon using a GitHub or Google account. Once authenticated, a Neon user account can create and manage projects, branches, users, databases, and other project resources.
 
+### NeonVM
+
+A QEMU-based tool used by Neon to create and manage VMs within a Kubernetes cluster, allowing for the allocation and deallocation of vCPU and RAM. For more information, refer to the [NeonVM](https://github.com/neondatabase/autoscaling/tree/main/neonvm) GitHub repository.
```

Comment:
I think technically this isn't a full githbub repo because it's in `neondatabase/autoscaling`, but I don't have any better ideas

Reply:
Revised.